pkg/repository/backend_postgres.go

```diff
@@ -1367,3 +1377,134 @@ func (r *PostgresBackendRepository) UpdateSecret(ctx context.Context, workspace
 
 	return &secret, nil
 }
+
+func (r *PostgresBackendRepository) CreateScheduledJob(ctx context.Context, scheduledJob *types.ScheduledJob) (*types.ScheduledJob, error) {
+	payloadJSON, err := json.Marshal(scheduledJob.Payload)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal scheduled job payload: %v", err)
+	}
+
+	// Remove invalid fields from schedule e.g. any field after 5th part
```

Comment:
can we add an example in the comment of what this looks like? not sure I follow

Reply:
Updated.